### DISCUSSION TITLE
Backup and restore history

### DISCUSSION THREAD

**Discussion by mmol67** [jellyfin/jellyfin#8761] [Q&A] [answered]
Can I backup my viewing history in one server and restore in another one? Just my viewing history.

**Comment by pypp** [comment, marked as answer]
Your viewing history is saved in 'UserDatas' table which containts :
key, userId rating, played, playCount, isFavorite, playbackPositionTicks, lastPlayedDate, AudioStreamIndex, SubtitleStreamIndex

If you want you can backup this table and restore it on another one manually, but it won't be only your viewing history.

**Reply by VlaK0r** [reply]
Tell me please, Where is this 'UserDatas'  table located?

**Reply by pypp** [reply]
For windows it's located at `C:\ProgramData\Jellyfin\Server\data\jellyfin.db`
